dev/bots/travis_upload.sh

```diff
@@ -17,3 +17,5 @@ fi
 
 # generate the API docs, upload them
 ./dev/bots/docs.sh
+
+coveralls-lcov $LCOV_FILE
```

Comment:
With this change, Travis will now be RED if coverage upload failed. I thought, that might be helpful. (Travis is ignoring any failures in `after_success`).